lib/glimesh/community_team.ex

```diff
@@ -0,0 +1,124 @@
+defmodule Glimesh.CommunityTeam do
+  @moduledoc """
+  The Community Team context :)
+  """
+  import Ecto.Query, warn: false
+  alias Glimesh.CommunityTeam.AuditLog
+  alias Glimesh.Repo
+
+  def access_level_to_title(level) do
+    case get_global_access_level(level) do
+      5 -> "Admin"
+      4 -> "Manager"
+      3 -> "Team Lead"
+      2 -> "Team Member"
+      1 -> "Trial Member"
+      _ -> "None"
+    end
+  end
+
+  def can_edit_user(user) do
+    if get_global_access_level(user.gct_level) >= 3, do: true, else: false
+  end
+
+  def can_edit_channel(user) do
+    if get_global_access_level(user.gct_level) >= 2, do: true, else: false
+  end
+
+  def can_edit_user_profile(user) do
+    if get_global_access_level(user.gct_level) >= 2, do: true, else: false
+  end
+
+  def can_ban_user(user) do
+    if get_global_access_level(user.gct_level) >= 2, do: true, else: false
+  end
+
+  def can_view_audit_log(user) do
+    if get_global_access_level(user.gct_level) >= 3, do: true, else: false
+  end
+
+  def can_view_billing_info(user) do
+    if get_global_access_level(user.gct_level) >= 4, do: true, else: false
+  end
+
+  def create_audit_entry(user, attrs \\ %{action: "None", target: "None"}) do
+    %AuditLog{
+      user: user
+    }
+    |> AuditLog.changeset(attrs)
+    |> Repo.insert()
+  end
+
+  def list_all_audit_entries(include_verbose? \\ false, params \\ []) do
+    _entries =
```

Comment:
_entires should be removed.

Reply:
Yeah I think at one point I was using the variable and just forgot to completely remove it.

Reply:
Still need to be removed.